apps/system/style/chrome/chrome.css

```diff
@@ -260,27 +260,48 @@
 
 
 /* SSL Lock Icons in rocketbar */
-.chrome.maximized .title[data-ssl="secure"] {
+.chrome-combined.maximized .chrome-has-ssl-indicator .chrome-ssl-indicator {
   -moz-padding-start: 3rem;
-  background: url("images/light/ssl.png") no-repeat 0.1rem center / 3rem 3rem;
+
+  background-repeat: no-repeat;
+  background-position: 0.1rem center;
+  background-size: 3rem;
 }
 
-.chrome.maximized .title[data-ssl="broken"] {
-  -moz-padding-start: 3rem;
-  background: url("images/light/ssl_broken.png") no-repeat 0.1rem center / 3rem 3rem;
+.private > .chrome-combined.maximized .chrome-has-ssl-indicator .chrome-ssl-indicator {
+  -moz-padding-start: 6.5rem;
+  background-position: 3.7rem 0.1rem;
 }
 
-.chrome.maximized .bar .title[data-ssl="secure"],
-.chrome.maximized .bar .title[data-ssl="broken"] {
-  text-indent: 2.5rem;
+.chrome-plain .chrome-ssl-indicator {
+  display: none;
+
+  width: 1rem;
+
+  background: no-repeat center / 3rem content-box;
 }
 
-.appWindow.private > .chrome.maximized .title[data-ssl="broken"],
-.appWindow.private > .chrome.maximized .title[data-ssl="secure"] {
-  -moz-padding-start: 6.5rem;
-  background-position: 3.7rem 0.1rem;
+.chrome-ssl-indicator-ltr {
+  padding-right: .5rem; /* give some space between the indicator and the title */
+}
+
+.chrome-ssl-indicator-rtl {
+  padding-left: .5rem; /* give some space between the indicator and the title */
+  padding-right: 1rem; /* give some space between the indicator and the window's edge */
+}
+
+.chrome-plain:-moz-dir(ltr) .chrome-has-ssl-indicator .chrome-ssl-indicator-ltr,
+.chrome-plain:-moz-dir(rtl) .chrome-has-ssl-indicator .chrome-ssl-indicator-rtl {
+  display: unset;
+}
+
+.chrome-has-ssl-indicator[data-ssl="secure"] .chrome-ssl-indicator {
```

Comment:
I think this should only be applied on chrome.maximized, otherwise you can see this in the minimized statusbar and it doesn't look right.